### PR TITLE
Add tooltip for bridged votes

### LINKED ISSUE #149: Add tooltip to clarify bridged votes
#### Description

- There may be a disconnect between the number of votes and the number of votes bridged because of bridging latency, Add  a tooltip to help clarify this for the user.

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description

- Add tooltip
- Add copy

Closes #149

### CODE REVIEW

**Comment by alexkeating** [comment]
Current dependencies on/for this PR:
* main
  * **PR #154** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/154?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #156** <a href="https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/156?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ScopeLift/l2-flexible-voting-frontend/156?utm_source=stack-comment).

File: components/ProposalRow.tsx [hunk 1]
```diff
@@ -69,6 +78,10 @@ export default function ProposalRow({
             )}
           ></div>
         </div>
+        <Tooltip
+          anchorSelect="#bridged-vote-explaination"
+          content="Bridged votes will not be reflected until the cross chain message has been relayed to the L1's chain which may take up to 25 minutes."
```
Comment (wildmolasses) [inline components/ProposalRow.tsx:83]: this is great. one thing i'm worried about: this copy doesn't mention that there's a manual bridging tx that must occur. Maybe they'll be misled to believe that their votes will automatically be bridged when they vote on L2 (w/ 25 min delay). do you think this is a concern?

**Review by wildmolasses — COMMENTED** [review]
lgtm, one comment

File: components/ProposalRow.tsx (hunk 1 above)
Comment (alexkeating) [reply components/ProposalRow.tsx:83]: Good question, definitely a concern for Wormhole testing. In the future some defender actions should handle this.  Open to a modification on the copy

Comment (alexkeating) [reply components/ProposalRow.tsx]: https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/156/commits/ecb56e1134eb780805ca695a09b76d107af279a6

**Pull request merged**